app/sim_test.go

```diff
@@ -248,8 +258,6 @@ func TestAppImportExport(t *testing.T) {
 		{app.keys[poststypes.StoreKey], newApp.keys[poststypes.StoreKey], [][]byte{}},
 		{app.keys[reportstypes.StoreKey], newApp.keys[reportstypes.StoreKey], [][]byte{}},
 		{app.keys[reactionstypes.StoreKey], newApp.keys[reactionstypes.StoreKey], [][]byte{}},
-
-		{app.keys[wasm.StoreKey], newApp.keys[wasm.StoreKey], [][]byte{wasmtypes.TXCounterPrefix}},
```

Comment:
It is because the `x/wasm` simulation does not generate the proper `LastInstanceID` for genesis state.  I created an issue on `wasmd`. I think we can add back after the issue is fixed.
https://github.com/CosmWasm/wasmd/issues/1372